src/devices/busch-jaeger.ts

```diff
@@ -139,6 +89,202 @@ const definitions: Definition[] = [
+        toZigbee: [tz.RM01_light_onoff_brightness, tz.RM01_light_brightness_step, tz.RM01_light_brightness_move],
+        onEvent: async (type, data, device) => {
+            const switchEndpoint = device.getEndpoint(0x12);
+            if (switchEndpoint == null) {
+                return;
+            }
+            // This device doesn't support reporting.
+            // Therefore we read the on/off state every 5 seconds.
+            // This is the same way as the Hue bridge does it.
+            if (type === 'stop') {
+                clearInterval(globalStore.getValue(device, 'interval'));
+                globalStore.clearValue(device, 'interval');
+            } else if (!globalStore.hasValue(device, 'interval')) {
+                const interval = setInterval(async () => {
+                    try {
+                        await switchEndpoint.read('genOnOff', ['onOff']);
+                        await switchEndpoint.read('genLevelCtrl', ['currentLevel']);
+                    } catch (error) {
+                        // Do nothing
+                    }
+                }, 5000);
+                globalStore.putValue(device, 'interval', interval);
+            }
+        },
+    },
+    {
+        fingerprint: [{modelID: 'RB01', endpoints: [{ID: 10}, {ID: 11}, {ID: 12}, {ID: 13}]}],
+        model: '6737/01',
+        vendor: 'Busch-Jaeger',
+        description: 'Zigbee Light Link wall-mounted transmitter 4rows',
+        endpoint: (device) => {
+            return {'row_1': 0x0a, 'row_2': 0x0b, 'row_3': 0x0c, 'row_4': 0x0d};
+        },
+        exposes: [e.action(['on_row_1', 'off_row_1', 'brightness_step_up_row_1', 'brightness_step_down_row_1', 'brightness_stop_row_1',
+            'on_row_2', 'off_row_2', 'brightness_step_up_row_2', 'brightness_step_down_row_2', 'brightness_stop_row_2',
+            'on_row_3', 'off_row_3', 'brightness_step_up_row_3', 'brightness_step_down_row_3', 'brightness_stop_row_3',
+            'on_row_4', 'off_row_4', 'brightness_step_up_row_4', 'brightness_step_down_row_4', 'brightness_stop_row_4'])],
+        meta: {multiEndpoint: true},
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint10 = device.getEndpoint(0x0a);
+            if (endpoint10 != null) {
+                await reporting.bind(endpoint10, coordinatorEndpoint, ['genLevelCtrl']);
+            }
+            const endpoint11 = device.getEndpoint(0x0b);
+            if (endpoint11 != null) {
+                const index = endpoint11.outputClusters.indexOf(5);
+                if (index > -1) {
+                    endpoint11.outputClusters.splice(index, 1);
+                }
+                await reporting.bind(endpoint11, coordinatorEndpoint, ['genLevelCtrl']);
+            }
+            const endpoint12 = device.getEndpoint(0x0c);
+            if (endpoint12 != null) {
```

Comment:
I think you can remove all these `if` checks on the endpoints, because the fingerprint will only match when the device has endpoint `12`